LICENSE.md

```diff
@@ -2,7 +2,7 @@ Original dagre-d3 copyright: Copyright (c) 2013 Chris Pettitt
 Original dagre copyright: Copyright (c) 2012-2014 Chris Pettitt
 Original graphlib copyright: Copyright (c) 2012-2014 Chris Pettitt
 
-Copyright (c) 2022-2023 Thibaut Lassalle and dagre-es contributors
+Copyright (c) 2022-2024 dagre-es contributors
```

Comment:
We could do:

```suggestion
Copyright (c) 2022-2024 Thibaut Lassalle, David Newell, Alois Klink, Sidharth Vinod and dagre-es contributors
```

Or, we could keep things simple, and just change the copyright year:

```suggestion
Copyright (c) 2022-2024 Thibaut Lassalle and dagre-es contributors
```

I believe both are fine from a legal standpoint.